auction9-frontend/src/app/data-table/data-table-details/data-table-details.component.ts

```diff
@@ -33,10 +45,18 @@ export class DataTableDetailsComponent implements OnInit {
       this.endTime = moment(this.auction.date_to).format("HH:mm");
       this.endDateTime = moment(this.endDate + ' ' + this.endTime).format("YYYY-MM-DD HH:mm");
     });
+
+    Storage.list(`${auctionId}`).then(data => {
+      data.forEach(res => {
+        // generate url
+        this.imageUrl = `https://auction9-auction-photos.s3-eu-west-1.amazonaws.com/public/${res.key}`;
```

Comment:
Bucket URL should be part of frontend configuration, so we can have separate bucket per environment.